frontend-react/src/pages/submissions/SubmissionDetails.test.tsx

```diff
@@ -77,6 +82,15 @@ describe("SubmissionDetails", () => {
             expect(i).toBeInTheDocument();
         }
     });
+
+    test("Filename conditionally shows in title", () => {
```

Comment:
LOVE the addition of tests!

Reply:
YES! Almost forgot, but getting into the habit of adding them :)